workflows-scripts/item-search-filter/jmeter-supported-data/credentials.csv

```diff
@@ -0,0 +1 @@
+[folio username],[foolio password],[tenantId]

```

Comment:
"foolio password" - FWIW there's a typo :)

Reply:
Not a typo. It's intentional because if I fix typo then that is a real password and we don't want to reveal it :)

Reply:
Or we can just do [username], [password], [tenantId]. I think it's self-explanatory what these creds are for.